modular_skyrat/master_files/code/game/objects/effects/decals/turfdecals/sr_markings.dm

```diff
@@ -0,0 +1,10 @@
+// Adds red variant that doesnt break 'bluesec' but allows mappers to use red elsewhere
```

Comment:
Why is it called `sr_markings.dm` if it's not really markings, nor is it senior markings?

Reply:
mostly did it to avoid duplicate naming